tls/ttls.c

```diff
@@ -2874,6 +2886,10 @@ ttls_init(void)
 	if (!ttls_hs_cache)
 		goto err_free;
 
+	if (tls_get_ja5_storage_size())
```

Comment:
In this PR we breaking project architecture introducing new cross-module dependencies: `tls` depends on `fw` and `tdb`, `lib` depends on `tdb`. I must point that modules loads in the order and these dependencies breaks the order. In `ja5_filter.c` we intermix helper functions and code that works with `tdb`, therefore each module who uses `lib` will depends on `tdb`, but `lib` must be independent.

I'm not sure that we must spread ja5 logic across the modules, looks like we can place this logic into `fw` module, then just configure `tls` as we do it now. I mean we must calculate rates and hit db relies on "events". We even can make new submodule `ja5` in `fw`(like `client` or `cache`). In lib we can place only function helpers that may be used from each module without new dependencies and completely stateless(now it's not).

Yes, I see that `ja5t` hash calculation is spread between `fw` and `tls`, but it's not looks so dangerous at least now, because we relies only on stateless, independent(means it not uses other modules) `lib` module here.

Reply:
done